src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -223,32 +225,14 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
         {
             ProcessKey? processKey = Utilities.GetProcessKey(pid, uid, name);
 
-            return InvokeForProcess(async processInfo =>
-            {
-                string dumpFileName = DumpUtilities.GenerateDumpFileName();
-
-                if (string.IsNullOrEmpty(egressProvider))
-                {
-                    Stream dumpStream = await _dumpService.DumpAsync(processInfo.EndpointInfo, type, HttpContext.RequestAborted);
```

Comment:
nit: Are we using `_dumpService` in this file anymore? If not, we probably don't need it anymore.

Reply:
Good catch. Removed it.